test/unit/matrix/test_matrix_hdf5.cpp

```diff
@@ -33,7 +33,7 @@ using dlaf::matrix::internal::FileHDF5;
 template <typename T>
 class MatrixHDF5Test : public ::testing::Test {
 protected:
-  MatrixHDF5Test() : world(MPI_COMM_WORLD), filepath("test_matrix_hdf5.h5") {
+  MatrixHDF5Test() : world(MPI_COMM_WORLD), filepath("/tmp/test_matrix_hdf5.h5") {
```

Comment:
Did you find out if there were restrictions on the directory where this test was run inside the CI job?

Reply:
Still experimenting. But running `ctest` directly on `daint` from `$HOME/git/DLA-Future/build` results in the same issue.